Jenkinsfile.download_logs

```diff
@@ -0,0 +1,67 @@
+pipeline {
+    agent { label 'download_logs' }
+
+    parameters {
+        string(name: 'REMOTE_SERVICE_URL', defaultValue: 'https://api.openshift.com', description: 'Service URL')
+        string(name: 'ASSISTED_INSTALLER_DEPLOYMENT_BRANCH', defaultValue: 'master', description: 'Branch on assisted-installer-deployment repo to use')
+        string(name: 'ASSISTED_TEST_INFRA_BRANCH', defaultValue: 'master', description: 'Branch on assisted-test-infra repo to use')
+    }
+
+    triggers { cron('H/30 * * * *') }
+
+    environment {
+        SKIPPER_PARAMS = " "
+
+        LOGS_DEST = "/var/ai-logs/triage_logs"
+
+        // Credentials
+        OFFLINE_TOKEN = credentials('admin_offline_token')
+        JIRA_ACCESS_TOKEN = credentials('assisted-installer-bot-jira-access-token')
+        SLACK_TOKEN = credentials('slack-token')
+    }
+    options {
+      timeout(time: 90, unit: 'MINUTES')
+    }
+
+    stages {
+        stage('Init') {
+            steps {
+                sh "rm -rf assisted-test-infra"
+                sh "git clone https://github.com/openshift-assisted/assisted-test-infra --branch ${ASSISTED_TEST_INFRA_BRANCH}"
```

Comment:
You'll need to fix this to openshift instead of openshift-assisted

Reply:
Of course, copy-paste mistake